### DISCUSSION TITLE
Can't even install RDT-client. Keeps crashing and cmd window closes immediately.

### DISCUSSION THREAD

**Discussion by cafesuapapi** [rogerfar/rdt-client#610] [General]
2.0.86

Windows 11 Home

Not using Docker

Real-Debrid

Google Chrome? Not sure how to answer that. 

[rdt-client error code.txt](https://github.com/user-attachments/files/17783941/rdt-client.error.code.txt)

**Comment by rogerfar** [comment]
If you look at the log you posted it says: `SQLite Error 26: 'file is not a database'.`. Did you configure your appsettings correctly?

**Comment by cafesuapapi** [comment]
I figured how to download it successfully. However, I noticed that it's constantly crashing afterwards. I did change the map and download path to /media/downloads and noticed that it wasn't crashing as bad as before but still crashes nonetheless. When I had everything defaulted after downloading, it crashed so bad I had to restart my computer several times. Do you know what the issue might be? Also noticed that sonarr would say "Local download client qBittorrent places downloads in \media\downloads\ but this is not a valid Windows path. Review your download client settings." but I'm using /media/downloads.
